kolibri/utils/cli.py

```diff
@@ -639,21 +639,31 @@ def _get_port(port):
     return int(port) if port else OPTIONS["Deployment"]["HTTP_PORT"]
 
 
+def _cleanup_before_quitting(signum, frame):
```

Comment:
what happens if there's a network outage or a sudden power off of the computer runing Kolibri? Is the announce of the service removed automatically, if not, how long is it kept?

Reply:
There's a 60 second TTL, though in practice it seems to take at least several minutes to "forget".

Reply:
And yes, it no longer announces no matter how it is killed. This is just to have it actively announce its departure so records are updated immediately.